main.py

```diff
@@ -491,3 +491,18 @@ def closeEvent(self, event):
     widget = Widget()
     widget.show()
     app.exec_()
+
```

Comment:
dont comment this info lines. revert this change. send it at pr description or comment on pr page.